<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnSetHeader">
          <property name="text">
           <string>设置表头</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnSetRows">
            <property name="text">
             <string>设置行数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnIniData">
          <property name="text">
           <string>初始化表格数据</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnInsertRow">
            <property name="text">
             <string>插入行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAppendRow">
            <property name="text">
             <string>添加行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btnDelCurRow">
          <property name="text">
           <string>删除当前行</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnResizeRow">
            <property name="text">
             <string>自动调节行高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnResizeCol">
            <property name="text">
             <string>自动调节列宽</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="btnReadToEdit">
          <property name="text">
           <string>读取表格内容到文本</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="chkBoxTabEditable">
            <property name="text">
             <string>表格可编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkBoxRowColor">
            <property name="text">
             <string>间隔行底色</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="chkBoxHeaderH">
            <property name="text">
             <string>显示行表头</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkBoxHeaderV">
            <property name="text">
             <string>显示列表头</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="9" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="rBtnSelectRow">
            <property name="text">
             <string>行选择</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rBtnSelectItem">
            <property name="text">
             <string>单元格选择</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableInfo"/>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQTableWidget">
    <property name="title">
     <string>QTableWidget</string>
    </property>
   </widget>
   <addaction name="menuQTableWidget"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
